source/Octopus.Client.Tests/Integration/HttpIntegrationTestBase.cs

```diff
@@ -91,7 +91,9 @@ private static X509Certificate2 GetCert()
             using (var ms = new MemoryStream())
             {
                 s.CopyTo(ms);
+#pragma warning disable PC001
```

Comment:
This is a warning we can safely ignore: `Integration\HttpIntegrationTestBase.cs(94,24): warning PC001: X509Certificate2.X509Certificate2(byte[], string) isn't supported on macOS [D:\dev\OctopusClients\source\Octopus.Client.Tests\Octopus.Client.Tests.csproj]`